.markdownlint.json

```diff
@@ -0,0 +1,4 @@
+{
+    "MD033" : false,
+    "MD013": false
+}
```

Comment:
Looks like you're running markdown lint here and have some config leaking into your PR.

We don't currently lint our markdown, maybe we should. Either way this should be removed from this PR. 

Maybe we should open an issue about linting markdown.

Reply:
My bad! Sorry!